Dockerfile

```diff
@@ -7,6 +7,7 @@ FROM node:${NODE_VERSION}-alpine AS development
 USER node
 WORKDIR /home/node
 
+ENV NODE_ENV development
```

Comment:
This is unnecessary as development is the default for `NODE_ENV`. We declare an `ARG` above here but don't actually use it until the production stage.